kinto/config/kinto.tpl

```diff
@@ -11,49 +13,101 @@ port = %(http_port)s
 #
-# Features.
-#
-# kinto.readonly = false
-# kinto.bucket_create_principals = system.Authenticated
+# kinto.readonly = False
 # kinto.batch_max_requests = 25
-
+# kinto.paginate_by =
 # Experimental JSON-schema on collection
-# kinto.experimental_collection_schema_validation = true
-
+# kinto.experimental_collection_schema_validation = False
 #
-# Plugins
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#activating-the-permissions-endpoint
+# kinto.experimental_permissions_endpoint = False
 #
+# kinto.trailing_slash_redirect_enabled = True
+# kinto.heartbeat_timeout_seconds = 10
+
+# Plugins
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#plugins
+# https://github.com/uralbash/awesome-pyramid
 kinto.includes = kinto.plugins.default_bucket
 #                kinto.plugins.admin
 #                kinto.plugins.accounts
 #                kinto.plugins.history
 #                kinto.plugins.quotas
 
+# Backends
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#storage
 #
-# Auth configuration.
+kinto.storage_backend = {storage_backend}
+kinto.storage_url = {storage_url}
+# kinto.storage_max_fetch_size = 10000
+# kinto.storage_pool_size = 25
+# kinto.storage_max_overflow = 5
+# kinto.storage_pool_recycle = -1
+# kinto.storage_pool_timeout = 30
+# kinto.storage_max_backlog = -1
+
+# Cache
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#cache
+#
+kinto.cache_backend = {cache_backend}
+kinto.cache_url = {cache_url}
+# kinto.cache_prefix = ''
+# kinto.cache_max_size_bytes = 524288
+# kinto.cache_pool_size = 25
+# kinto.cache_max_overflow = 5
+# kinto.cache_pool_recycle = -1
+# kinto.cache_pool_timeout = 30
+# kinto.cache_max_backlog = -1
+
+# Permissions.
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#permissions
 #
+kinto.permission_backend = {permission_backend}
+kinto.permission_url = {permission_url}
+# kinto.permission_pool_size = 25
+# kinto.permission_max_overflow = 5
+# kinto.permission_pool_recycle = 1
+# kinto.permission_pool_timeout = 30
+# kinto.permission_max_backlog - 1
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#bypass-permissions-with-configuration
+# kinto.bucket_create_principals = system.Authenticated
+
+# Scheme, host, and port
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#scheme-host-and-port
+#
+# kinto.http_host
+# kinto.http_scheme
+
+# Authentication
 # https://kinto.readthedocs.io/en/latest/configuration/settings.html#authentication
 #
 kinto.userid_hmac_secret = {secret}
 multiauth.policies = basicauth
-# multiauth.policies = fxa basicauth
-
+# Any pyramid multiauth setting can be specified for custom authentication
+# https://github.com/uralbash/awesome-pyramid#authentication
 #
-# Accounts API configuration.
+# Accounts API configuration
 #
 # Enable built-in plugin.
 # kinto.includes = kinto.plugins.accounts
```

Comment:
I wonder if we should repeat the setting name. The danger is that someone would just uncomment this line, and it will override the `kinto.includes` that we gave above.

Maybe, instead we could just add a sentence instead:  
```
# Add `kinto.plugins.accounts` to `kinto.includes` setting
```

What do you think?